_posts/en/newsletters/2025-08-01-newsletter.md

```diff
@@ -0,0 +1,244 @@
+    [taproot][topic taproot], simplifying implementation for any
+    software that already supports taproot.
+
+  - It commits to the taproot [annex][topic annex], which `OP_CTV` does
+    not.  One way this can be used is to ensure some data is published
+    as part of a transaction, such as data used in a contract protocol
+    to allow a counterparty to recover from publication of an old state.
+
+  - It redefines an `OP_SUCCESSx` opcode rather than an `OP_NOPx`
+    opcode.  Soft forks redefining `OP_NOPx` opcodes must be `VERIFY`
+    opcodes that mark the transaction invalid if they fail.  Redefinitions
+    of `OP_SUCCESSx` opcodes can simply place either `1` (success) or
+    `0` (failure) on the stack after execution; this allows them to be
+    used directly in cases where redefined `OP_NOPx` redefinitions would
+    need to be wrapped by conditionals such as `OP_IF`
+    statements.
+
+  - "It prevents surprising inputs with ... `scriptSig`."  FIXME:harding
+    summarize and link to bitVM discussion
+
+  Brandon Black [replied][black th] with a comparison of the proposal to
+  his earlier LNHANCE bundle proposal (see [Newsletter #285][news285
+  ik]) and found it comparable in most ways, although he noted that it
+  is less efficient in onchain space for _congestion control_ (a form of
+  delayed [payment batching][topic payment batching]).
+
+- **Proposal to allow longer relative timelocks:** developer Pyth
+  [posted][pyth timelock] to Delving Bitcoin to suggest allowing
+  [BIP68][] relative timelocks to be extended from their current maximum
+  of about one year to a new maximum of about ten years.  This would
+  require a soft fork and the use of an additional bit from the
+  transaction input _sequence_ field.
+
+  Fabian Jahr [replied][jahr timelock] with a concern that
+  [timelocks][topic timelocks] too far in the future could lead to a
+  loss of funds, such as due to the development of quantum computers
+  (or, we add, the deployment of quantum defense protocols such as
+  Jameson Lopp's proposal described earlier in this newsletter).  Steven
+  Roose [noted][roose timelock] that far-future timelocks are already
+  possible using other time lock mechanisms (such as presigned
+  transactions and [BIP65 CLTV][bip65]), and Pyth added that their
+  desired use case is for a wallet recovery path where the long timelock
+  would only be used in the primary path became unavailable and the
+  alternative would be permanent loss of the funds anyway.
+
+- **Security against quantum computers with taproot as a commitment scheme:**
+  Tim Ruffing [posted][ruffing qtr] a link to a [paper][ruffing paper]
+  he wrote analyzing the security of [taproot][topic taproot]
+  commitments against manipulation by quantum computers.  He examines
+  whether taproot commitments to tapleaves would continue to posses the
+  _binding_ and _hiding_ properties it posses against classical
```

Comment:
```suggestion
  _binding_ and _hiding_ properties it possesses against classical
```
Or maybe "it has against" (just to avoid two "possess" so close together).